cli/tests/integrations/test_package.py

```diff
@@ -256,7 +263,12 @@ def test_describe_options():
     actual_labels = stdout_.pop("labels", None)
 
     for label, value in expected_labels.items():
-        assert value == actual_labels.get(label)
+        if label == "DCOS_PACKAGE_METADATA":
+            # We can covert the metadata into a dictionary
```

Comment:
Could you rephrase this comment to explain why converting into a dictionary is a good idea? The fact that we're doing the conversion is clear from the code, but it's less clear why we would want to.